Dockerfile.helm-unit-test

```diff
@@ -21,6 +21,9 @@ RUN wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64
 RUN mkdir -p /conjur-authn-k8s-client
 WORKDIR /conjur-authn-k8s-client
 
+# Workaround for CVE-2022-24765 when running git inside a docker container
```

Comment:
Not sure if this is the only way around this issue as of yet but due to a git security fix, the container needs to explicitly run a safe.directory command before running.